spec/services/distribution_fetch_itemized_breakdown_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+RSpec.describe DistributionFetchItemizedBreakdownService, type: :service, skip_seed: true do
+  describe ".fetch(as_csv: false)" do
+    subject { service.fetch(as_csv: as_csv) }
+    let(:service) { described_class.new(organization: organization, distribution_ids: distribution_ids) }
+    let(:organization) { create(:organization) }
+    let(:distribution_ids) { distributions.pluck(:id) }
+    let(:distributions) { create_list(:distribution, 2, :with_items, item_quantity: distribution_per_item, organization: organization) }
+    let(:distribution_per_item) { 50 }
+    let(:as_csv) { false }
+    let(:expected_output) do
+      distributions.map(&:items).flatten.each_with_object({}) do |item, acc|
+        acc[item.name] ||= {}
+        acc[item.name] = {
+          distributed: distribution_per_item,
+          current_onhand: InventoryItem.find_by(item_id: item.id).quantity,
+          onhand_minimum: item.on_hand_minimum_quantity,
+          below_onhand_minimum: item.on_hand_minimum_quantity > InventoryItem.find_by(item_id: item.id).quantity
```

Comment:
I know that a lot of specs rely on this sort of behavior, but I'd like to see us hardcode expected values as much as possible. If it turns out we don't know what the value would be, that's an indication that our spec is relying too much on randomized / faker data.